### PR TITLE
Versioning changes for PR #6069

### PR DESCRIPTION
**Pull request by praries880**
Versioning changes for PR #6069

### CODE REVIEW

**Comment by praries880** [comment]
sign job : https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/706/

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/908

File: src/ResourceManager/Storage/AzureRM.Storage.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '5.0.2'
+ModuleVersion = '5.0.3'
```
Comment (maddieclayton) [inline src/ResourceManager/Storage/AzureRM.Storage.psd1]: I think you'll probably want to bump the version of profile to the current version on the gallery (5.1.0) - line 54

Comment (praries880) [reply src/ResourceManager/Storage/AzureRM.Storage.psd1]: done...

**Comment by praries880** [comment]
merged preview into the branch, do not need PR #6369 anymore

**Comment by praries880** [comment]
Travis build seems to have succeeded but was terminated. I just kicked it again.

**Comment by praries880** [comment]
kicked off a new sign build and on demand as well

Sign : https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/709

on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/913

**Comment by praries880** [comment]
new sign job : https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/710/

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/914/

**Comment by praries880** [comment]
@blueww 
The nuget package will be under : https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/710/artifact/src/Package/ 
you will need the AzureRM.Storage and AzureRM.Profile pkgs from thereblueww

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/915/

**Comment by praries880** [comment]
new sign job : https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/712/

**Pull request merged**